### ISSUE TITLE
Keepass like key search?

### ISSUE THREAD

**Issue by 3k2** [keepassxreboot/keepassxc#1955]
in Keepass on windows if you click in the program any entry you have and just start typing example "go" it will highlight entry named google I'd love to have this behaviour in keepassxc on linux. When I do the same in keepassxc it will highlight the entry "go" starting but to select it properly you have to click or use arrow keys to navigate one down or up then back to the entry you originally wanted in first place.

**Comment by droidmonkey** [comment]
Do you mean outside of a search? As in you have an entry selected in the list and just start typing.

**Comment by 3k2** [comment]
Yeah, exactly outside of search. In keepass if you do that it properly highlights the entry but in keepassxc it highlights the entry by those outlines but not properly until you press arrow down or up.

**Comment by droidmonkey** [comment]
The fix for this might also apply to #960

**Comment by droidmonkey** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#1955 - Search current entry list just by typing (like file explorer)](https://trello.com/c/hflssPB4/13-1955-search-current-entry-list-just-by-typing-like-file-explorer)

**Comment by droidmonkey** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#1955 - Search current entry list just by typing (like file explorer)](https://trello.com/c/hflssPB4/13-1955-search-current-entry-list-just-by-typing-like-file-explorer)

**Comment by Invertisment** [comment]
As an alternative it would be great to use TAB button to focus search results

**Comment by droidmonkey** [comment]
Press down arrow

**Comment by Invertisment** [comment]
I already know that and I used it as my only options. 
But as my fingers are on my home row I wanted to find an ergonomic way to do it. 
And reaching for the arrows is not ergonomic. And I liked how keepass2 allowed to use TAB to select it.

Ideally I would like to have something emacs-like Ctrl+N and Ctrl+P. Which would act as up/down arrows.

**Comment by droidmonkey** [comment]
Actually, sorry TAB does work... as long as you are using a recent version of KeePassXC (2.6.0+). SHIFT+TAB as well. Also F3!
 https://github.com/keepassxreboot/keepassxc/pull/4641
